sherlock_project/resources/data.json

```
@@ -121,6 +121,12 @@
     "urlMain": "https://discussions.apple.com",
     "username_claimed": "jason"
   },
+  "Aparat": {
+    "errorType": "status_code",
+    "url": "https://www.aparat.com/{}/",
+    "urlMain": "https://www.aparat.com/",
+    "username_claimed": "jadi"
```

This returns true for all tested items (false positives).
The endpoint used always returns 200 or a 304 to a 200, even though there are 404s found in the stream elsewhere.

The API endpoint it uses in-stream _does_ directly return a 404, however.

Additionally, `status_code` requests default to `HEAD` rather than `GET`, which the api endpoint doesn't like.

This should be resolved by adding the following items:

```suggestion
    "errorType": "status_code",
    "request_method": "GET",
    "url": "https://www.aparat.com/{}/",
    "urlMain": "https://www.aparat.com/",
    "urlProbe": "https://www.aparat.com/api/fa/v1/user/user/information/username/{}",
    "username_claimed": "jadi"
```

---

You can accept this directly by hitting the Commit suggestion button, or you can edit locally and push.